src/web/routes/figma/integration.test.ts

```diff
@@ -140,7 +152,7 @@ describe('/figma', () => {
 		describe('FILE_UPDATE event', () => {
 			let connectInstallation: ConnectInstallation;
 			let figmaTeam: FigmaTeam;
-			let associatedFigmaDesigns: AssociatedFigmaDesign[];
```

Comment:
We were using this previously in tests to generate `AtlassianDesign`s / `FileResponse`s. This array was populated as we upserted `AssociatedFigmaDesign` records in the `beforeEach` block:

```typescript
associatedFigmaDesigns.push(
	await associatedFigmaDesignRepository.upsert(
		associatedFigmaDesignCreateParams,
	),
```

This was causing tests to fail when I switched over to using `FigmaService.fetchDesignsByIds`, because the order of `associatedFigmaDesigns` now needed to match the order of associated designs returned by `AssociatedFigmaDesignRepository.findManyByFileKeyAndConnectInstallationId`, so that `nock` can correctly match query params / request body.

To fix this, I now query `associatedFigmaDesigns` from the database in each test instead.